compiler/noirc_frontend/src/hir/resolution/visibility.rs

```diff
@@ -7,34 +7,42 @@ use std::collections::BTreeMap;
 use crate::ast::ItemVisibility;
 use crate::hir::def_map::{CrateDefMap, DefMaps, LocalModuleId, ModuleId};
 
-// Returns false if the given private function is being called from a non-child module, or
-// if the given pub(crate) function is being called from another crate. Otherwise returns true.
-pub fn can_reference_module_id(
+/// Returns true if an item with the given visibility in the target module
+/// is visible from the current module. For example:
+///
+/// mod foo {
+///     ^^^ <-- target module
+///   pub(crate) fn bar() {}
+///   ^^^^^^^^^^ <- visibility
+/// }
```

Comment:
```suggestion
/// ```text
/// mod foo {
///     ^^^ <-- target module
///   pub(crate) fn bar() {}
///   ^^^^^^^^^^ <- visibility
/// }
/// ```
```